turbo/execution/eth1/forkchoice.go

```diff
@@ -284,12 +285,20 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 			}
 		}
 
+		unwindToGenesis = e.executionPipeline.UnwindPoint() == 0
```

Comment:
Sure its correct? Genesis is committed with txnum=1 and txn=0 carries allocations.
So unwind down to 0 will re-exec genesis and then continue execution, as of my understanding. 
Those start-end transactions are applied during execution while in unit tests we do not expect those 2 txs so this raises questions like that.

Reply:
This is kind of an hotfix as unwind to genesis just does not work